addons/lookatrandom/addon.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Point towards random direction",
```

Comment:
This needs a better name.

Reply:
We could say "Randomize Direction Option", or something similar.

Reply:
Maybe "Random direction option in point towards block"?

Scratch Addons is somewhat picky about addon names, so I'm really not sure.

Reply:
I'm just randomly hopping into this discussion, but I think the original name is the best (or maybe add "block" to the end of it)